src/Resources/config/doctrine/BaseMedia.orm.xml

```diff
@@ -16,7 +16,7 @@
         <field name="height" column="height" type="integer" nullable="true"/>
         <field name="length" column="length" type="decimal" nullable="true" precision="10"/>
         <field name="contentType" column="content_type" type="string" nullable="true" length="255"/>
-        <field name="size" column="content_size" type="integer" nullable="true"/>
+        <field name="size" column="content_size" type="integer" length="5" nullable="true"/>
```

Comment:
Should we use an even bigger length ?

Reply:
There is an internal mapping inside doctrine where length > 4 maps to BIGINT